xwiki-platform-core/xwiki-platform-model/xwiki-platform-model-validation/xwiki-platform-model-validation-api/src/main/java/org/xwiki/model/validation/EntityNameValidation.java

```diff
@@ -67,4 +67,16 @@
      * @return {@code true} if the policy is respected.
      */
     boolean isValid(EntityReference entityReference);
+
+    /**
+     * Clean the configuration cache of a wiki.
+     *
+     * @param wikiId the identifier of the wiki's cache to clean (e.g., {@code "wikiA"})
+     * @since 13.4
+     * @since 12.10.8
+     */
+    default void cleanConfigurationCache(String wikiId)
```

Comment:
hmm this doesn't look very good to me. This seems to be a management API and that should go in a JMX MBean. It's definitely wrong to me to have that in a "business" interface such as EntityNameValidation.